src/components/usertask-component/form-fields/enum-form-field.tsx

```diff
@@ -22,7 +22,7 @@ export class EnumFormField implements IFormField {
   }
 
   public componentWillLoad(): void {
-    this.value = this.formField.defaultValue;
+    this.value = this.formField.defaultValue || this.formField.enumValues[0].id;
```

Comment:
Wenn ich den PR richtig verstehe, soll das hier ein Null-Check darstellen, richtig?

In dem Fall wäre `??` der bessere Operator.

```suggestion
    this.value = this.formField.defaultValue ?? this.formField.enumValues[0].id;
```

Reply:
defaultValue könnte null/undefined, oder ein leerer String sein. Deshalb `||`, der prüft ja auch auf `''`

Reply:
Ja. Aber der Operator prüft z.B. auch auf `true` bzw. `false` und `0` bzw. `1`. Das könnte zu false positives führen, wenn ich ein boolean als default value verwenden will.

Vorschlag:

```suggestion
    this.value = this.formField.defaultValue?.length > 0 || this.formField.enumValues[0].id;
```

Reply:
Ja das sind strings vom BPMN Studio. Wenn man nichts eingetragen hat ist halt null/undefined, aber wenn jemand hingeht und was einträgt und nachher wieder ausm Feld was löscht dann ist es ein `''`.

Reply:
Wir können aber auch um ganz sicher zu gehen dass niemand es schafft da nen anderen type einzugeben z.b. boolean oder number. Dann auf 

```suggestion
    this.value = this.formField.defaultValue != null &&  this.formField.defaultValue !== '' ?  this.formField.defaultValue : this.formField.enumValues[0].id;
```

machen